dotcom-rendering/src/lib/edition.ts

```diff
@@ -1,63 +1,106 @@
 import { isOneOf } from '@guardian/libs';
-import type { EditionLinkType } from '../model/extract-nav';
+import { isTuple, type Tuple } from './tuple';
 
-export type EditionId = (typeof editionList)[number]['editionId'];
+type EditionId = 'UK' | 'US' | 'AU' | 'INT' | 'EUR';
 
-export const editionList = [
+type Edition = {
+	url: string;
+	editionId: EditionId;
+	pageId: string;
+	longTitle: string;
+	title: string;
+	dateLocale: string;
+	timeZone: string;
+	langLocale?: string;
+	hasEditionalisedPages: boolean;
+};
+
+const editionList: Tuple<Edition, 5> = [
 	{
 		url: '/preference/edition/uk',
 		editionId: 'UK',
+		pageId: 'uk',
 		longTitle: 'UK edition',
 		title: 'UK edition',
-		locale: 'en-gb',
+		dateLocale: 'en-gb',
 		timeZone: 'Europe/London',
+		langLocale: 'en-GB',
+		hasEditionalisedPages: true,
 	},
 	{
 		url: '/preference/edition/us',
 		editionId: 'US',
+		pageId: 'us',
 		longTitle: 'US edition',
 		title: 'US edition',
-		locale: 'en-us',
+		dateLocale: 'en-us',
 		timeZone: 'America/New_York',
+		langLocale: 'en-US',
+		hasEditionalisedPages: true,
 	},
 	{
 		url: '/preference/edition/au',
 		editionId: 'AU',
+		pageId: 'au',
 		longTitle: 'Australia edition',
 		title: 'AU edition',
-		locale: 'en-au',
+		dateLocale: 'en-au',
 		timeZone: 'Australia/Sydney',
+		langLocale: 'en-AU',
+		hasEditionalisedPages: true,
 	},
 	{
 		url: '/preference/edition/int',
 		editionId: 'INT',
+		pageId: 'international',
 		longTitle: 'International edition',
 		title: 'International edition',
-		locale: 'en-gb',
+		dateLocale: 'en-gb',
 		timeZone: 'Europe/London',
+		langLocale: 'en',
+		hasEditionalisedPages: false,
 	},
 	{
 		url: '/preference/edition/eur',
 		editionId: 'EUR',
+		pageId: 'europe',
 		longTitle: 'Europe edition',
 		title: 'Europe edition',
-		locale: 'en-gb',
+		dateLocale: 'en-gb',
 		timeZone: 'Europe/Paris',
+		hasEditionalisedPages: false,
 	},
+];
+
+const ukEdition = editionList[0];
```

Comment:
If you’re the destructuring afficionado…

```suggestion
const [ ukEdition ] = editionList;
```